task-12/04-ansible/inventory.ini

```diff
@@ -0,0 +1,7 @@
+[webservers]
```

Comment:
We should consider using aws_ec2 inventory plugin instead 
https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_ec2_inventory.html

Reply:
example: aws_inventory.yml
```
plugin: aws_ec2
regions:
  - "eu-central-1"
keyed_groups:
  - prefix: ec2
    key: tags.Terraform
  - prefix: name
    key: tags.Name
filters:
  instance-state-name : running
compose:
  ansible_host: public_ip_address

```